app/views/layouts/application.html.erb

```diff
@@ -107,6 +107,10 @@
     <% end %>
     <%= yield(:script) if content_for?(:script) %>
     <!-- hiding intercom on hub again until we tackle styling it into the sticky action footer -->
-    <%= render("shared/intercom", isHub: hub?) unless (hub? || Routes::CtcDomain.new.matches?(request) || Routes::StateFileDomain.new.matches?(request)) %>
+    <% if state_file? %>
+      <%= render("shared/state_file_intercom", current_intake: current_intake, state: params[:us_state] )%>
```

Comment:
I'm curious does this work if we don't pass in `current_intake` and `params[:us_state]`?

Reply:
you're right i don't need to pass it in!